OpenRA.Mods.Common/Traits/Air/Aircraft.cs

```diff
@@ -157,7 +157,7 @@ public void UnReserve()
 			}
 		}
 
-		public void Killed(Actor self, AttackInfo e)
+		public void Disposing(Actor self)
 		{
 			UnReserve();
```

Comment:
The trait isn't being disposed - the actor that owns it is.

The problem with keeping this in `Killed` is that the reservation isn't removed if the actor is removed from the world without being explicitly killed (e.g. by selling it or using Lua).